### ISSUE TITLE
Penis uncircumcised

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9871]
**Preferred term label:**
Penis uncircumcised

**Synonyms**
Foreskin present, prepuce present

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Balanitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal Adding new term: 
- ID: HP:6000296
- label: Penis uncircumcised
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: PMID:25011345
- def: A medical history in a male of not having been circumcized.
- comment: Neonatal male circumcision is a common surgical procedure.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9871
- synonyms: Foreskin present(NONE)|Prepuce present(NONE)
